tests/testthat/test-endpoints-model.R

```diff
@@ -218,15 +218,15 @@ test_that("model run can be cancelled", {
   testthat::try_again(5, {
     Sys.sleep(1)
     log <- queue$queue$worker_log_tail(worker, n = Inf)
-    expect_true("INTERRUPT" %in% log$command)
-    expect_equal(queue$queue$task_status(id), setNames("INTERRUPTED", id))
+    expect_true("CANCEL" %in% log$command)
+    expect_equal(queue$queue$task_status(id), setNames("CANCELLED", id))
   })
 
   get_status <- queue_status(queue)
   response <- get_status(id)
   expect_true(response$done)
   expect_false(response$success)
-  expect_equal(response$status, scalar("INTERRUPTED"))
+  expect_equal(response$status, scalar("CANCELLED")) # TODO: update schema?
```

Comment:
Old TODO here, looks like status is just a `string` in schema so I don't think anything to change.